receiver/splunkhecreceiver/config.go

```diff
@@ -21,6 +22,9 @@ type Config struct {
 	confighttp.ServerConfig `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
 	splunk.AccessTokenPassthroughConfig `mapstructure:",squash"`
+
+	Ack `mapstructure:",squash"`
```

Comment:
Why do we need to squash and have all the options prefixed with `ask_`? Why not keep it as a config group?
```yaml
ack:
  extension:
  path:
```